src/neo/SmartContract/ApplicationEngine.Binary.cs

```diff
@@ -1,11 +1,14 @@
 using Neo.VM.Types;
+using static System.Convert;
 
 namespace Neo.SmartContract
 {
     partial class ApplicationEngine
     {
         public static readonly InteropDescriptor System_Binary_Serialize = Register("System.Binary.Serialize", nameof(BinarySerialize), 0_00100000, TriggerType.All, CallFlags.None, true);
         public static readonly InteropDescriptor System_Binary_Deserialize = Register("System.Binary.Deserialize", nameof(BinaryDeserialize), 0_00500000, TriggerType.All, CallFlags.None, true);
+        public static readonly InteropDescriptor System_Binary_Base64Encode = Register("System.Binary.Base64Encode", nameof(Base64Encode), 0_00100000, TriggerType.All, CallFlags.None, true);
```

Comment:
Maybe we can create a new namespace for conversions?

Reply:
If there are more conversions in the future, consider creating a namespace. Not needed yet.

Reply:
Later it's hard to change it because it could be used by SmartContracts.

Reply:
Before mainnet launches.